ascon-hash/Cargo.toml

```diff
@@ -19,15 +19,16 @@ rust-version = "1.56"
 
 [dependencies]
 digest = { version = "0.10", default-features = false, features = ["core-api"] }
-ascon = "0.3"
+ascon = { version = "0.4", default-features = false }
 
 [dev-dependencies]
 spectral = { version = "0.6", default-features = false }
 hex = "0.4"
 
 [features]
-default = ["std"]
+default = ["std", "zeroize"]
```

Comment:
Usually we do not enable `zeroize` by default.

Reply:
Okay, in this case I will remove it :+1: